src/Outsystems.SetupTools/Functions/Set-OSServerConfig.ps1

```diff
@@ -129,6 +133,12 @@ function Set-OSServerConfig
                     $paramDictionary.Add('LogDBCredential', $LogDBCredentialParam)
                 }
             }
+            $InstallServiceCenterAttrib = New-Object System.Management.Automation.ParameterAttribute
+            $InstallServiceCenterAttrib.ParameterSetName = 'ApplyConfig'
+            $InstallServiceCenterAttribCollection = New-Object System.Collections.ObjectModel.Collection[System.Attribute]
+            $InstallServiceCenterAttribCollection.Add($InstallServiceCenterAttrib)
+            $InstallServiceCenterParam = New-Object System.Management.Automation.RuntimeDefinedParameter('InstallServiceCenter', [switch], $ConfigureCacheInvalidationServiceAttribCollection)
```

Comment:
$ConfigureCacheInvalidationServiceAttribCollection ?

Reply:
fixe, missed that one